internal/acceptance_test/resource_caas_machine_blueprint_test.go

```diff
@@ -0,0 +1,136 @@
+package acceptancetest
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/HewlettPackard/hpegl-containers-go-sdk/pkg/mcaasapi"
+
+	"github.com/HewlettPackard/hpegl-containers-terraform-resources/pkg/auth"
+	"github.com/HewlettPackard/hpegl-containers-terraform-resources/pkg/client"
+)
+
+const (
+	// Fill in these values based on the environment being used for acceptance testing
+	nameMbp         = "mbp-test"
+	siteIDMbp       = ""
+	machineProvider = "vmaas"
+	osImage         = "sles-custom"
+	osVersion       = ""
+	computeType     = ""
+	size            = ""
+	storageType     = ""
+)
+
+var machineRoles = []string{"controlplane"}
+
+// nolint: gosec
+func testCaasMachineBlueprint() string {
+
+	return fmt.Sprintf(`
+	provider hpegl {
+		caas {
+			api_url = "https://client.greenlake.hpe.com/api/caas/mcaas"
+		}
+	}
+	data "hpegl_caas_site" "blr" {
+		name = "BLR"
+		space_id = "%s"
+	  }
+	resource hpegl_caas_machine_blueprint test {
+		name         = "%s"
+  		site_id = data.hpegl_caas_site.blr.id
+  		machine_roles = ["%s"]
+		machine_provider = %s
+        os_image = %s
+        os_version = %s
+        compute_type = %s
+        size = %s
+        storage_type = %s
```

Comment:
Could you format this file (lines 50-54)

Reply:
in my Dev VM, it is coming as formatted. Don't know why it is coming like this after pushing.

Reply:
Fixed.